log/logger.go

```diff
@@ -74,6 +74,20 @@ func NewLogger(dst io.Writer, options ...Option) Logger {
 	for _, opt := range options {
 		opt(&logCfg)
 	}
+	if logCfg.JSONMarshal != nil {
```

Comment:
Any chance we can remove the init: https://github.com/cosmos/cosmos-sdk/blob/03fb7da/log/logger.go#L14-L27 now that it is configurable?

Reply:
I agree with you. Also init function will change `zerolog.InterfaceMarshalFunc`'s value once we import `"cosmossdk.io/log"`, I have remove it and set the default `JSONMarshal` to `json.Marshal` in `func NewLogger` for backport

Reply:
Since `github.com/rs/zerolog` only support for setting global json marshaler that would affect all instances of zerolog.Logger, we shouldn't make it configurable in "Logger instance level". But we can make it configurable in "cosmossdk.io/log" package level.

Reply:
Hi @julienrbrt , I have updated the new implementation of json marshaler customization~